### DISCUSSION TITLE
Find file name when not fullscreen on desktop monitor?

### DISCUSSION THREAD

**Discussion by peteruithoven** [Rafostar/clapper#296] [Q&A] [answered]
How does one figure out the file name / media title of the file that's currently being played? The only way seems to be to go fullscreen on a big enough monitor. On a mobile screen (width < 1280) there is no way? 

![Screenshot from 2022-07-10 12-07-11@2x](https://user-images.githubusercontent.com/523210/178140444-0658de40-2279-4987-80b2-5bef539a437b.png)
 
The info is visible fullscreen on bigger screens:
![Screenshot from 2022-07-10 12-08-57@2x](https://user-images.githubusercontent.com/523210/178140496-e3d86d3a-c4f3-4448-ba26-58b6a7c72a47.png)

**Comment by peteruithoven** [comment, marked as answer]
You can click on the time/duration at the bottom to see the playlist which includes the file name.

**Reply by Rafostar** [reply]
The mobile UI was done like that to not cover too much of the small screen its normally used on (as seen in e.g. [this video](https://youtu.be/BvmRV6IIGGo)). You using it on laptop is a little unexpected to be honest :smile: 

Slightly related, there is an issue open here with the idea to integrate reading media info and using title from that (if any) to show it as title in the playlist instead of current filename/path, which should make it easier to read.

**Reply by peteruithoven** [reply]
> The mobile UI was done like that to not cover too much of the small screen its normally used on (as seen in e.g. [this video](https://youtu.be/BvmRV6IIGGo)).

I think that makes sense. 

I think one challenge is that people don't think the current time / duration is clickable and or don't think to look there for a playlist, playback speed etc. People even opened 2 issues (#121 and #271) because they didn't think to look there. I opened started looking into that whole mobile screen issue #291 because I didn't. 
But I'd prefer to have a good suggestion before starting a discussion on that.
